src/user-comment/controllers/user-comment.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Body, Controller, Get, Param, Post, Req } from '@nestjs/common';
+import { UserCommentService } from '../service/user-comment.service';
+import { CommentDto } from '../dto/comment.dto';
+import { Comments } from '../entities/Comments.entity';
+import { ApiTags } from '@nestjs/swagger';
+
+@Controller('comment')
+@ApiTags('Comment')
+export class UserCommentController {
+  constructor(private readonly commentsService: UserCommentService) {}
+  @Get('allComments/:postId')
```

Comment:
rest standarts are not kept at all.

Reply:
I attempted to fix that. I hope it's correct now.